NOTES.md

```diff
@@ -0,0 +1,2 @@
+# NOTES
+- TBD
```

Comment:
While the Issue Description on Github lists `add NOTES with notes about commenting out default values in config`, the committed file holds no information.

Reply:
Fair. Thanks